### ISSUE TITLE
[5.2.0-beta.2]: `Getting unknown property: craft\fields\Dropdown::0`

### ISSUE THREAD

**Issue by tommysvr** [craftcms/cms#15147]
### Description
Getting `Getting unknown property: craft\fields\Dropdown::0` when querying for entries with a dropdown field

### Steps to reproduce

1. Add a Dropdown field to your Entry Type
2. `{% set entries = craft.entries().dropdownField('foo').all() %}`
3.  Error as above

### Expected behavior
Able to query for entries with dropdown field

### Actual behavior
```
yii\base\UnknownPropertyException: Getting unknown property: craft\fields\Dropdown::0 in /var/www/html/vendor/yiisoft/yii2/base/Component.php:154
Stack trace:
#0 /var/www/html/vendor/yiisoft/yii2/base/Model.php(1034): yii\base\Component->__get('0')
#1 /var/www/html/packages/cms/src/elements/db/ElementQuery.php(2587): yii\base\Model->offsetGet(0)
#2 /var/www/html/packages/cms/src/elements/db/ElementQuery.php(1566): craft\elements\db\ElementQuery->_loopInCustomFields()
#3 /var/www/html/vendor/yiisoft/yii2/db/QueryBuilder.php(227): craft\elements\db\ElementQuery->prepare(Object(craft\db\mysql\QueryBuilder))
#4 /var/www/html/vendor/yiisoft/yii2/db/Query.php(157): yii\db\QueryBuilder->build(Object(craft\elements\db\EntryQuery))
#5 /var/www/html/vendor/yiisoft/yii2/db/Query.php(249): yii\db\Query->createCommand(Object(craft\db\Connection))
#6 /var/www/html/packages/cms/src/db/Query.php(256): yii\db\Query->all(NULL)
#7 /var/www/html/packages/cms/src/elements/db/ElementQuery.php(1778): craft\db\Query->all(NULL)
#8 /var/www/html/vendor/twig/twig/src/Extension/CoreExtension.php(1635): craft\elements\db\ElementQuery->all()
#9 /var/www/html/packages/cms/src/helpers/Template.php(141): twig_get_attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(craft\elements\db\EntryQuery), 'all', Array, 'method', false, false, false, 1)
#10 /var/www/html/storage/runtime/compiled_templates/15/159b33f743e82245dd0d8c28250ade05.php(39): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(craft\elements\db\EntryQuery), 'all', Array, 'method', false, false, false, 1)
#11 /var/www/html/vendor/twig/twig/src/Template.php(394): __TwigTemplate_0207b0bb5c75d26c7fa9e84b453dd1fc->doDisplay(Array, Array)
#12 /var/www/html/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#13 /var/www/html/vendor/twig/twig/src/Template.php(379): Twig\Template->display(Array)
#14 /var/www/html/vendor/twig/twig/src/TemplateWrapper.php(38): Twig\Template->render(Array)
#15 /var/www/html/vendor/twig/twig/src/Environment.php(280): Twig\TemplateWrapper->render(Array)
#16 /var/www/html/packages/cms/src/web/View.php(482): Twig\Environment->render('dropdown', Array)
#17 /var/www/html/packages/cms/src/web/View.php(535): craft\web\View->renderTemplate('dropdown', Array)
#18 /var/www/html/packages/cms/src/web/TemplateResponseFormatter.php(57): craft\web\View->renderPageTemplate('dropdown', Array, 'site')
#19 /var/www/html/vendor/yiisoft/yii2/web/Response.php(1109): craft\web\TemplateResponseFormatter->format(Object(craft\web\Response))
#20 /var/www/html/packages/cms/src/web/Response.php(338): yii\web\Response->prepare()
#21 /var/www/html/vendor/yiisoft/yii2/web/Response.php(340): craft\web\Response->prepare()
#22 /var/www/html/vendor/yiisoft/yii2/base/Application.php(390): yii\web\Response->send()
#23 /var/www/html/web/index.php(12): yii\base\Application->run()
#24 {main}
```

### Craft CMS version

5.2.0-beta.2

### PHP version

8.2

**Comment by i-just** [comment]
Thanks for reporting, Tommy! I raised a PR for this.

**Comment by brandonkelly** [comment]
Fixed in 5.2.0-beta.3. Thanks again @tommysvr!
